docs/governance/grants/grants-program.md

```diff
@@ -0,0 +1,93 @@
+2. **Impact for the CoW ecosystem**: The positive impact that a grant will have on the overall CoW Protocol ecosystem is a crucial factor in the evaluation process.
+   The potential effects on user adoption, community growth, platform scalability, and ecosystem expansion will be carefully considered.
+   Grants that demonstrate the ability to significantly enhance the ecosystem's functionality, user base, or value proposition will be prioritized.
+3. **Sanity check**: The sanity check criterion ensures that the workload estimation for the proposed grant is accurate and reasonable.
+   The evaluation process will verify if the grant application provides a well-defined scope, timeline, and resource requirements.
+   It aims to assess whether the proposed workload aligns with the expected outcomes and whether the grantee has adequately considered the feasibility and practicality of the project.
+4. **Technical expertise of the grantee(s)**:  The technical expertise of the grantee(s) is an important aspect of the evaluation process.
+  It assesses whether the individuals or team behind the grant application possess the necessary skills, knowledge, and experience to successfully deliver the proposed grant milestones.
+  The evaluation will consider their track record, expertise in relevant technologies, and the ability to execute the project effectively.
+
+By considering these judgment criteria, the CoW Grants Program aims to ensure that grants awarded align with the objectives of CoW Protocol and contribute to its growth and development.
+Grants that demonstrate feasibility, have a positive impact on the ecosystem, provide realistic workloads, and are backed by competent individuals or teams will have higher chances of receiving funding and support.
+
+# Committee Members and Safe Control Structure
+
+As determined in [CIP-8](https://snapshot.org/#/cow.eth/proposal/0xdc641be107f139753cea051f1bacb8b74b915713a95306c3994f5e03e20d6bef), the CoW Grants Program Safe is controlled by two signers. Safe #3 is shared with the CoW DAO (Safe #1) and the Grants Committee (Safe #2). Safe #3 holds all the grant funds. You can check the governing process in the image below:
+
+(Todo: Create a mermaid chart of Safes including committee, grants-funds, volume-grants, cow-dao)
+
+The grants committee will have a distribution of 8 community members, making a total of 7 committee members. Safe #2 (from the schematic image above) [`gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD`](https://app.safe.global/settings/setup?safe=gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD) will function as the Grants committee Safe and has 8 signers with a signing threshold of 5 out of 8. The specified committee members:
```

Comment:
Confirm the number of committee / community members

Reply:
Are there 8 community members?

Reply:
IMO this becomes increasingly irrelevant, we can just say the number of committee memebers

Reply:
If it's irrelevant, or seemingly irrelevant, we can remove it. It's just that "8 community members, making a total of 7 committee members" doesn't make sense?